pkg/api/pod/util.go

```diff
@@ -842,3 +844,14 @@ func multiplePodIPsInUse(podStatus *api.PodStatus) bool {
 	}
 	return false
 }
+
+// GetPodPriority returns priority of the given pod.
```

Comment:
This looks like the wrong location for this function:
It should probably be `pkg/api/v1/pod`

And there could be an equivalent here for the type `api.Pod`. But if it's not currently useful, don't add it.

Reply:
+1, this should be in pkg/api/v1/pod/util.go

Reply:
> Will review once the tests are passing.
You can take a look now